docs/user/production-considerations/reporting-production-considerations.asciidoc

```diff
@@ -28,14 +28,15 @@ For an additional layer of security, use the sandbox. The Chromium sandbox uses
 [[reporting-linux-sandbox]]
 ==== Linux sandbox
 The Linux sandbox depends on user namespaces, which were introduced with the 3.8 Linux kernel. However, many
-distributions don't have user namespaces enabled by default, or they require the CAP_SYS_ADMIN capability. The {report-features}
-automatically disable the sandbox when it is running on Debian because additional steps are required to enable
-unprivileged usernamespaces. In these situations, you'll see the following message in your {kib} startup logs:
+distributions don't support or allow non-privileged processes to create user namespaces, or they require the CAP_SYS_ADMIN capability. The {report-features}
+automatically disable the sandbox when it detects it is running on systems where it is not enabled.
+Without explicitly setting `xpack.screenshotting.browser.chromium.disableSandbox: false` in `kibana.yml`,
+the {report-features} may detect that it can't be enabled. In these situations, you'll see the following message in your {kib} startup logs:
```

Comment:
```suggestion
the {report-features} may detect that it can't be enabled. In the event it is automatically disabled, you'll see the following message in your {kib} startup logs:
```